web/cobrands/hackney/assets.js

```diff
@@ -0,0 +1,275 @@
+(function(){
+
+if (!fixmystreet.maps) {
+    return;
+}
+
+/** These layers are from the Hackney WFS feed, for non-Alloy categories: */
+var wfs_defaults = {
+  http_options: {
+    url: "https://map.hackney.gov.uk/geoserver/wfs",
+    params: {
+        SERVICE: "WFS",
+        VERSION: "1.1.0",
+        REQUEST: "GetFeature",
+        SRSNAME: "urn:ogc:def:crs:EPSG::27700"
+    }
+},
+  asset_type: 'spot',
+  max_resolution: 2.388657133579254,
+  asset_id_field: 'id',
+  attributes: {},
+  geometryName: 'geom',
+  srsName: "EPSG:27700",
+  strategy_class: OpenLayers.Strategy.FixMyStreet,
+  body: "Hackney Council",
+  asset_item: "item"
+};
+
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "greenspaces:park_bin",
+      }
+  },
+  asset_category: "Overflowing bin",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "amenity:public_toilet",
+      }
+  },
+  asset_category: "Public toilets",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "parking:pay_display_machine_liberator",
+      }
+  },
+  asset_category: "Pay & Display Machines",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "recycling:estate_recycling_bin",
+      }
+  },
+  asset_category: "Bin Contamination",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "amenity:sport_facility",
+      }
+  },
+  asset_category: "Sport Area",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "greenspaces:park_bench",
+      }
+  },
+  asset_category: "Park Furniture (bench)",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "greenspaces:park_bin",
+      }
+  },
+  asset_category: "Park Furniture (bin)",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "greenspaces:park_picnic_bench",
+      }
+  },
+  asset_category: "Park Furniture (picnic bench)",
+  attributes: {}
+});
+
+fixmystreet.assets.add(wfs_defaults, {
+  http_options: {
+      params: {
+          TYPENAME: "amenity:public_toilet",
+      }
+  },
+  asset_category: "Toilets",
```

Comment:
This appears identically again further down, same asset and category.
Plus can it not be combined with "Public toilets"?